static/app/gettingStartedDocs/react-native/react-native.tsx

```diff
@@ -227,12 +198,6 @@ const onboarding: OnboardingConfig<PlatformOptions> = {
               {
                 language: 'javascript',
                 code: getConfigureSnippet(params),
-                additionalInfo: tct(
```

Comment:
Also removed this part, since it does not make sense to mention 'sentry-wizard' in the manual installation. 

![Screenshot 2025-03-14 at 07 06 25](https://github.com/user-attachments/assets/e50a5caa-8db2-40f3-ac6b-a661d8f9a47d)